viewing_party/party.py

```diff
@@ -0,0 +1,240 @@
+
+#Thank you for the patience and understanding!
+
+movies = []
+watchlist =[]
+watched = []
+
+
+
+def create_movie(movie_title, genre, rating):
+    #create_movie_no_title_returns_none
+    #create_movie_no_genre_returns_none
+    #create_movie_no_genre_returns_none
+
+    if (movie_title == None) or (genre == None) or (rating == None):
+        return None 
+
+    else:
+        new_movie = {}
+        new_movie["title"] = movie_title
+        new_movie["genre"] = genre
+        new_movie["rating"] = rating
```

Comment:
I recommend building all of this in one go.  It saves typing and makes things a bit clearer:

```suggestion
        new_movie = {
            "title": movie_title,
            "genre": genre,
            "rating": rating
        }
```